web/src/main/webapp/WEB-INF/pages/application_state.jsp

```diff
@@ -22,6 +22,8 @@
         <link href="${bootstrap_datepicker_css}" rel="stylesheet" type="text/css">
         <script src="${moment_js}" type="text/javascript"></script>
         <script src="${bootstrap_datetimepicker_js}" type="text/javascript"></script>
+
+        <fmt:setTimeZone value="Europe/Budapest" scope="session"/>
```

Comment:
Pleases put this one level higher in the jsp hierarchy, so all pages have this setting by default. Also I think that scope attribute can be left at default (page), because we are reapplying this setting on every pageload anyway by having this code in the JSP.